src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -55,30 +58,52 @@ public void createFeedbackResponse(FeedbackResponseAttributes fra)
             }
         }
     }
-
+    
     public FeedbackResponseAttributes getFeedbackResponse(
             String feedbackResponseId) {
-        return frDb.getFeedbackResponse(feedbackResponseId);
+        FeedbackResponseAttributes response = frDb.getFeedbackResponse(feedbackResponseId);
+        if (response == null) {
+            return null;
+        }
+        ArrayList<FeedbackResponseAttributes> responses = new ArrayList<FeedbackResponseAttributes>();
```

Comment:
To standardize, make `responses` a `List` instead of `ArrayList` (this is also known as 'loose coupling').